teacher.rb

```diff
@@ -1,8 +1,10 @@
 require_relative 'person'
 
 class Teacher < Person
-  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
-    super(age, name, parent_permission)
+  attr_reader :age, :specialization, :name
+
+  def initialize(age, specialization, name, id, parent_permission: true)
```

Comment:
- [ ]  I can not add teacher, kindly fix the error
![image](https://github.com/LoviMadaan/oop-school-library/assets/66097371/0483090e-08d7-476e-b87d-a5126e04cb5a)